content/id/docs/concepts/storage/volumes.md

```diff
@@ -0,0 +1,1151 @@
+
+Pengetahuan akan [Pods](/docs/user-guide/pods) disarankan.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Latar Belakang
+
+Docker juga memiliki konsep _[volume]_(https://docs.docker.com/engine/admin/volumes/), walaupun konsepnya Docker agak lebih longgar dan kurang dikelola. Pada Docker, sebuah volume adalah sesederhana sebuah direktori pada _disk_ atau  di dalam Container lainnya. _Lifetime_ tidak dikelola dan hingga baru-baru ini hanya ada volume yang didukung _disk_ lokal. Docker sekarang menyediakan _driver_ untuk volume, namun fungsionalitasnya masih sangat terbatas (misalnya hingga Docker 1.7 hanya ada satu _driver_ volume yang diizinkan untuk setiap Container, dan tidak ada cara untuk menyampaikan parameter kepada volume).
+
+Sebaliknya, sebuah Volume Kubernetes memiliki _lifetime_ yang gamblang - sama dengan _lifetime_ Pod yang berisi Volume tersebut. Oleh karena itu, sebuah Volume bertahan lebih lama dari Container-container yang berjalan di dalam Pod tersebut, dan data di Volum tersebut juga dipertahankan melewati diulangnya Container. Tentu saja, saat sebuah Pod berakhir, Volume tersebut juga akan berakhir/terhapus. Dan mungkin lebih penting lagi, Kubernetes mendukung banyak jenis Volume, dan sebuah Pod dapat menggunakan sebanyak apapun Volume secara bersamaan.
+
+Pada intinya, sebuah volume hanyalah sebuah direktori, dan mungkin berisi data, yang dapat diakses oleh Container-container di dalam Pod. Bagaimana direktori tersebut dibuat, medium yang menyokongnya, dan isinya ditentukan oleh jenis volume yang digunakan.
+
+Untuk menggunakan sebuah volume, sebuah Pod memerinci volume-volume yang akan disediakan untuk Pod tersebut (kolom `.spec.volumes`) dan di mana volume-volume tersebut akan ditambatkan (di-_mount_) di dalam Container-container di Pod (kolom `.spec.containers.volumeMounts`).
+
+Sebuah proses di dalam Container memiliki sudut pandang _filesystem_ yang disusun dari _image_ dan volume Dockernya. [Docker Image](https://docs.docker.com/userguide/dockerimages/) berada pada bagian teratas hierarki _filesystem_, dan volume manapun yang ditambatkan pada _path_ yang diperinci di dalam Image tersebut. Volume tidak dapat ditambatkan pada volume lain atau memiliki _hard link_ ke volume lain. Setiap Container di dalam Pod harus secara independen memerinci di mana tiap Volume ditambatkan.
```

Comment:
```suggestion
Sebuah proses di dalam kontainer memiliki sudut pandang _filesystem_ yang disusun dari _image_ dan volume Dockernya. [Docker Image](https://docs.docker.com/userguide/dockerimages/) berada pada bagian teratas hierarki _filesystem_, dan volume manapun yang ditambatkan pada _path_ yang diperinci di dalam Image tersebut. Volume tidak dapat ditambatkan pada volume lain atau memiliki _hard link_ ke volume lain. Setiap kontainer di dalam Pod harus secara independen memerinci di mana tiap Volume ditambatkan.
```
Should make sure `mount` translation is consistent.

Reply:
Yups, there are several:

Mount (verb) -> menambatkan
Mounted (passive participle) -> ditambatkan
Mount (noun) -> tambatan

Reply:
I chose it as it is the one that makes sense in Indonesian.
But there are two things that I translate into "Menambatkan":

- Attach (verb)
- Mount (verb)

I'm starting to think that it might cause confusions later. Or should we just italicize them instead? What do you think?

@irvifa @girikuncoro @qbl @danninov

Reply:
"tambat" is not standard in mas ivan lanin's glosarium yet, so probably better to stick with italicize. But I'm fine with "tambat" as long as we include parentheses as well to help educate the readers
```
... tambat (_mount_)
```

Reply:
but if this is too much effort, feel free to stick on what you have translated

Reply:
The "mount" (verb and noun) occurs very frequently in the docs, and I find it strange leaving it as is, it's confusing when it is a verb, and when it is a noun, mixed between Indonesian sentences.
So I decided to use this translation effort, and find it less confusing.
I think I'll add the parenthesis at the first occurrence of the verb, and also the adjective, and stick on it.